cocos/editor-support/spine/AnimationState.c

```diff
@@ -414,7 +415,8 @@ float _spAnimationState_applyMixingFrom (spAnimationState* self, spTrackEntry* e
 		}
 	}
 
-	_spAnimationState_queueEvents(self, from, animationTime);
+	if (entry->mixDuration > 0) _spAnimationState_queueEvents(self, from, animationTime);
+	internal->eventsCount = 0;
```

Comment:
@natural-law , I didn't find this changes in the link (https://github.com/EsotericSoftware/spine-runtimes/commit/3ced1e1daf722793740130094b9352e07b3dced7) you mentioned.

Did you paste a wrong url? I also compared the code in spine runtime and found there are not this changes in 3.5.35 spine runtime.

Reply:
Sorry for the wrong information. The correct related commit is:
https://github.com/EsotericSoftware/spine-runtimes/commit/c512ee5d595076842cbb12d95a3a7f06792b81d1